packages/core/src/codewhisperer/util/customizationUtil.ts

```diff
@@ -94,20 +94,11 @@ export const baseCustomization = {
  * @returns customization selected by users, `baseCustomization` if none is selected
  */
 export const getSelectedCustomization = (): Customization => {
-    if (
-        !AuthUtil.instance.isCustomizationFeatureEnabled ||
-        !AuthUtil.instance.isValidEnterpriseSsoInUse() ||
-        !AuthUtil.instance.conn
-    ) {
+    if (!AuthUtil.instance.isCustomizationFeatureEnabled || !AuthUtil.instance.isIdcConnection()) {
         return baseCustomization
     }
 
-    const selectedCustomizationArr = globals.globalState.tryGet<{ [label: string]: Customization }>(
-        'CODEWHISPERER_SELECTED_CUSTOMIZATION',
-        Object,
-        {}
-    )
-    const selectedCustomization = selectedCustomizationArr[AuthUtil.instance.conn.label]
+    const selectedCustomization = globals.globalState.getCodewhispererCustomization(AuthUtil.instance.profileName)
```

Comment:
cw is a legacy name isn't it? the key name doesn't matter
```suggestion
    const selectedCustomization = globals.globalState.getAmazonQCustomization(AuthUtil.instance.profileName)
```